tutorials/julia-jupyter-notebook-server/index.md

```diff
@@ -0,0 +1,138 @@
+* acme_certificate.certificate: check that the *common_name* field contains the correct FQDN for your notebook server
+* acme_registration.reg: check that the *email_address* field contains the correct email address
+* google_compute_firewall.jupyter-server: check that the *target_tags.nnnnnnnnnn* field is set to *jupyter-server-[your server name]*
+* google_compute_instance.nbs_acme_cert: check that the *tags.nnnnnnnnnn* field is set to *jupyter-server-[your server name]*
+* google_dns_record_set.nbs_acme_cert: check that the *name* field contains the correct FQDN for your notebook server
+* tls_private_key.private_key
+
+If you set ```use_acme_cert``` to false only these resources will be created/configured
+* google_compute_firewall.jupyter-server: check that the *target_tags.nnnnnnnnnn* field is set to *jupyter-server-[your server name]*
+* google_compute_instance.nbs_self_signed_cert: check that the *tags.nnnnnnnnnn* field is set to *jupyter-server-[your server name]*
+* google_dns_record_set.nbs_self_signed_cert: check that the *name* field contains the correct FQDN for your notebook server
+
+## Create the notebook server
+To create the notebook server type
+```sh
+terraform apply tf.plan
+```
+The terminal output logs Terraform's progress as it executes the plan you generated earlier. When it completes you will see:
+```sh
+Apply complete! Resources: 6 added, 0 changed, 0 destroyed.
+```
+Installing the Juila kernel takes approximately 10 minutes. Therefore even though the Compute Engine instance is running you won't be able to immediately connect to the notebook server. Wait 10 minutes and then proceed.
+
+## Log into the notebook server
+The URL for your notebook server has the form: ```https://[your server name].[your domain]:8089```, *e.g.*, ```https://dobbs.sierramadre.net:8089```. When you navigate there with your browser you should see the Jupyter login screen.
+
+![Jupyter Login Screen](https://github.com/wardharold/community/blob/master/tutorials/julia-jupyter-notebook-server/secure-jupyter-login.png)
+
+Note that if your notebook server has a self-signed certificate your browser will complain that the connection is not private because it does not recognize the self-signed certificate. You will need to manually accept the certificate to proceed.
+
+Enter your password and click the ```Log in``` button and you should see Jupyter interface.
+
+![Jupyter Interface](https://github.com/wardharold/community/blob/master/tutorials/julia-jupyter-notebook-server/secure-jupyter-interface.png)
+
+The ```setup.sh``` script clones a set of Julia [tutorials](https://github.com/JuliaComputing/JuliaBoxTutorials) from the [Julia Computing](https://juliacomputing.com/) GitHub [repo](https://github.com/JuliaComputing) to get started.
+
+## Clean up
+When you've finished your work teardown the notebook server, delete the firewall rule, and remove the DNS records associated with the notebook server.
```

Comment:
"When you've finished your work teardown the notebook server, delete the firewall rule, and remove the DNS records associated with the notebook server." -> "When you've finished your work, tear down the notebook server, delete the firewall rule, and remove the DNS records associated with the notebook server."

Reply:
fixed in Update index.md (18)